src/modules/RelocationServer.rb

```diff
@@ -233,7 +234,16 @@ def Read
       if !ReadLibvirtServices()
         Report.Error(_("Cannot read the current libvirtd/sshd state."))
         Report.Error(Message.CannotContinueWithoutPackagesInstalled)
-        return false
+        inst_hyper = Popup.YesNo(
+          _("Run 'Install Hypervisor and Tools' to correct this?")
+          )
+        if inst_hyper == true
```

Comment:
you can omit the comparison operator and just check inst_hyper and even do a 

```ruby
return false unless Popup.YesNo(_("Run 'Install Hypervisor and Tools' to correct this?"))
```

But what about the state of ssh, it is not read in this case.

Apart of that ConfigureDom0 needs like a cleanup, huge method with definition of methods inside of it.. I will maybe help refactoring it a little bit in the future and adding some unit tests.